operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -0,0 +1,432 @@
+  |     | o- lun26  [fileio/7c0bba5c5301c97 (/var/lib/cps-local/boot-images/5b43428e-4381-4f39-9335-6dababb76d86/rootfs) (default_tg_pt_gp)]
+  |     | o- lun27  [fileio/380840014ffe295 (/var/lib/cps-local/boot-images/f731d8d5-0fed-41d7-996e-6a0d19b6ff6d/rootfs) (default_tg_pt_gp)]
+  |     o- portals .................................................................................................... [Portals: 3]
+  |       o- 10.102.104.28:3260 ............................................................................................... [OK]
+  |       o- 10.150.0.4:3260 .................................................................................................. [OK]
+  |       o- 10.252.1.13:3260 ................................................................................................. [OK]
+  o- loopback ......................................................................................................... [Targets: 0]
+  o- vhost ............................................................................................................ [Targets: 0]
+  o- xen-pvscsi ....................................................................................................... [Targets: 0]
+```
+
+In the above `targetcli ls` command output, we can see that four `fileio` backing store are created for
+two `rootfs` images and two iSCSI `LUNs` are created which has the `rootfs` image id being mapped. And there
+are 26 `PE` or `squashfs` `fileio` backing store created and 26 iSCSI `LUNs` created which have `PE`
+or `squashfs` image id being mapped. Both of these iSCSI `LUNs` are ready for projection.
+
+Sample initiator node snippet after the projection:
+
+```bash
+nid001032:~ # multipath -ll
+11218.831779 | /etc/multipath.conf line 10: ignoring deprecated option "disable_changed_wwids", using built-in value: "yes"
+PE_CPE-base.x86_64-24.11.squashfs (36001405de4cc04e7dacfb9ada0a6b4cc) dm-0 LIO-ORG,de4cc04e7dacfb9
+size=7.7G features='1 queue_if_no_path' hwhandler='1 alua' wp=ro
+`-+- policy='round-robin 0' prio=50 status=active
+  |- 1:0:0:14 sdo  8:224  active ready running
+  |- 2:0:0:14 sdaq 66:160 active ready running
+  |- 3:0:0:14 sdbs 68:96  active ready running
+  `- 4:0:0:14 sdcu 70:32  active ready running
+f731d8d5-0fed-41d7-996e-6a0d19b6ff6d_rootfs (36001405380840014ffe295091e8689db) dm-24 LIO-ORG,380840014ffe295
+size=11G features='1 queue_if_no_path' hwhandler='1 alua' wp=ro
+`-+- policy='round-robin 0' prio=50 status=active
+  |- 1:0:0:27 sdab 65:176 active ready running
+  |- 2:0:0:27 sdbd 67:112 active ready running
+  |- 3:0:0:27 sdcf 69:48  active ready running
+  `- 4:0:0:27 sddh 70:240 active ready running
+```
+
+## Steps to achieve SBPS
+
+1. Node personalisation
+2. Run GOSS Test suite
+3. Creation of BOS session template
+4. Image tagging
+5. Boot compute / UAN's
+6. Monitor iSCSI Metrics
+
+### Node Personalisation (Worker node)
```

Comment:
We should probably use the US spelling of Personalization rather than the British English spelling (i.e. Personalisation) just to be consistent with other parts of the documentation.

Reply:
Sure :)

Reply:
There are 5 other instances of [Pp]ersonalisation, please also fix them.

Reply:
Done